packages/adapter-vercel/index.js

```diff
@@ -307,6 +307,11 @@ async function create_function_bundle(builder, entry, dir, runtime) {
 	traced.warnings.forEach((error) => {
 		// pending https://github.com/vercel/nft/issues/284
 		if (error.message.startsWith('Failed to resolve dependency node:')) return;
+		
+		// parse errors are likely not js and can safely be ignored,
+		// such as this html file in "main" meant for nw instead of node:
+		// https://github.com/vercel/nft/issues/311
+		if (error.message.startsWith('Unexpected token')) return;
```

Comment:
I think it should be this?

```suggestion
		if (error.message.startsWith('Failed to parse')) return;
```